### ISSUE TITLE
Gallery breaks when dragging tab to new window in Firefox

### ISSUE THREAD

**Issue by DJW90** [dimsemenov/PhotoSwipe#987]
Hi,

Let me start off by saying my knowledge on this subject is limited and the problem could be solely on my end. Having said that I have noticed a possible bug/issue with Photoswipe. Lets say you have multiple pages open as tabs on your browser, like google.com, photoswipe,com, etc. You want photoswipe.com to be open in a window on its own so you click and drag the tab so it opens in a new window. However, after it opens as a new window and you click on one of the thumbnail images nothing happens. The cursor changes to the magnifying glass, so the browser seems to think that it opened one of the images but the gallery doesn't launch. Also if you hover your cursor to upper right corner, where the controls should be, the cursor changes to the pointer, which would further suggest that the browser thinks that gallery is open. Once you have done this none of links work and you have to refresh the page in order to get the page functional again. 

This issue only affects the gallery. If you you drag the tab into a new window and click any of the links on the bottom (Documentation, GitHub, Twitter) the page will work properly. Also if you drag the tab from one window to anther the gallery works fine. In other words you have google.com open in one window, and github.com + photoswipe.com open in another. Then you drag the photoswipe.com tab from one into into the other so you have google.com + Photoswipe.com open in one window and only github open in the other.

I have tested this in Chrome, Ie, and Safari, but it seems like Firefox is the only browser that has this issue. 

Has anyone else encountered this? Is this issue on my end?

PS I'm currently using Firefox 41.0.1.

**Comment by dimsemenov** [comment]
- Do you test on Windows or Mac?
- Do you have any extensions in FF?

I can't repeat this for some reason. Should the gallery be open when its tab is dragged?

**Comment by DJW90** [comment]
It seems like the FF extensions were the problem. I disabled all of them and tested it and it worked fine. Then I tried enabling them one by one so I could narrow down which one was causing the problem, but I got to the end of the list and I can't seem to recreate the bug anymore. 

Just to answer your questions and to have as a record I was testing on Windows for Chrome, IE, FF and Mac for Safari.

The extensions I had were: Adblock Plus, Adblock Plus Pop-up, Awesome Screenshot, DownThemAll!, Flash Video Download, Ghostery, Private Tab, User Agent Switcher, Video Download Helper, and YSlow.

Thanks for the help!
